appveyor.yml

```diff
@@ -0,0 +1,16 @@
+clone_folder: C:\projects\neon-lang\
```

Comment:
According to https://www.appveyor.com/docs/build-configuration#clone-directory the clone directory default is indeed `c:\projects\whatever`. The next paragraph says *if* it's required by your project, then you can explicitly specify the clone directory. Neon should not have any absolute path references, therefore it is not necessary to ask for a specific clone directory.